flipperdevices/flipperzero-firmware
# fbt: added support for FBT_NO_SYNC

# What's new

- If FBT_NO_SYNC environment variable is present, fbt skips submodule update at startup

# Verification 

- Check out any submodule to a different revision
- Try building with `fbt` when FBT_NO_SYNC is set or not

# Checklist (For Reviewer)
